### PR TITLE
DOC/CLN: fixed typos in timeseries.rst

### PR DESCRIPTION
**Pull request by mortada**

### CODE REVIEW

File: doc/source/timeseries.rst [hunk 1]
```diff
@@ -1204,7 +1204,7 @@ As the same as `DatetimeIndex`, the endpoints will be included in the result. Be
 
 Frequency Conversion and Resampling with PeriodIndex
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-The frequency of Periods and PeriodIndex can be converted via the ``asfreq``
+The frequency of `Periods` and `PeriodIndex` can be converted via the ``asfreq``
```
Comment (shoyer) [inline doc/source/timeseries.rst]: I think you want to use two backticks here around "Periods" (which should actually be "Period") and "PeriodIndex"

Comment (mortada) [inline doc/source/timeseries.rst]: ah good catch ... i found a few other places where it should be double backticks instead of single

**Comment by mortada** [comment]
@shoyer it's updated

**Comment by shoyer** [comment]
thanks!

**Pull request merged**